harness/determined/deploy/aws/templates/efs.yaml

```diff
@@ -761,6 +761,7 @@ Resources:
                   ssh_key_name: ${Keypair}
                   tag_key: det-${AWS::StackName}
                   tag_value: det-agent-${AWS::StackName}
+                  cpu_slots: true
```

Comment:
I think this field naming is confusing. To me it seemed like `cpu_slots=True` is forcing slots to use CPUs, but it looks like the field is actually specifying whether to use CPU slots if GPUs aren't available. IMO it needs a clearer name.

The new field should also be documented.

Reply:
changed to `cpu_slots_allowed` and updated the docs.